Makefile

```diff
@@ -283,6 +290,8 @@ release: clean-release ## Builds and push container images using the latest git
 
 .PHONY: release-manifests
 release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a release
+	$(KUSTOMIZE) build config/core > $(RELEASE_DIR)/core-components.yaml
+	$(KUSTOMIZE) build config/kubeadm > $(RELEASE_DIR)/bootstrap-components.yaml
```

Comment:
Should this be broken out a bit more granular into separate components for the bootstrap provider and for the control plane provider here?

Reply:
oh sure, probably. hang tight and I'll split those up too.